docs/configuration.md

```diff
@@ -2456,7 +2456,8 @@ Apart from these, the following properties are also available, and may be useful
     For more detail, see the description
     <a href="job-scheduling.html#dynamic-resource-allocation">here</a>.
     <br><br>
-    This requires <code>spark.shuffle.service.enabled</code> to be set.
+    This requires <code>spark.shuffle.service.enabled</code> or
+    <code>spark.dynamicAllocation.shuffleTracking.enabled</code> to be set.
```

Comment:
`spark.shuffle.service.enabled=false` and `spark.dynamicAllocation.shuffleTracking.enabled=true` is experimental now? https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala#L208

If so, its better to leave some notes about that here?

Reply:
You mean documenting that dynamic allocation without a shuffle service is experimental? I think it's fine as. that's documented at `spark.dynamicAllocation.shuffleTracking.enabled` at https://github.com/apache/spark/pull/24817/files#diff-76e731333fb756df3bff5ddb3b731c46R2121, and it will show a warning as you pointed out.

Reply:
ok